Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build
+FROM golang:latest AS base
```

Comment:
What's the rationale behind the `base` stage?

Reply:
None. I just copy-paste stuff because I don't know how what it does. I've removed it now ;)